### DISCUSSION TITLE
Iterators for N = 0 iterations

### DISCUSSION THREAD

**Discussion by dariosanfilippo** [grame-cncm/faustlibraries#116] [Ideas]
Hello again.

Following my last discussion on the sorting algorithm, for the implementation, it was necessary to define a signal bus returning

0 : !

for a bus of N = 0 signals.

This discussion has taken place before but I don't quite remember the details of it; are there any cases where it wouldn't make sense for the iterators to return 

0 : !

for N = 0 iterations?

Ciao,
Dario

**Comment by magnetophon** [comment]
Yeah, I've needed this various times as well.
https://github.com/grame-cncm/faustlibraries/issues/101

**Comment by sletz** [comment]
What do you think @orlarey  ? Should we do that for `par, seq, sum, prod`operators ? https://faustdoc.grame.fr/manual/syntax/#iterations
